node/metrics/metrics.go

```diff
@@ -21,6 +24,104 @@ type StatusHandlersUtils interface {
+	appStatusHandler.SetUInt64Value(common.MetricNonce, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricCountConsensus, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricCountLeader, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricCountAcceptedBlocks, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumTxInBlock, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumMiniBlocks, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumProcessedTxs, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricCurrentRoundTimestamp, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricHeaderSize, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricMiniBlocksSize, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumShardHeadersFromPool, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumShardHeadersProcessed, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumTimesInForkChoice, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricHighestFinalBlock, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricCountConsensusAcceptedBlocks, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricRoundAtEpochStart, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNonceAtEpochStart, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricRoundsPassedInCurrentEpoch, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNoncesPassedInCurrentEpoch, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricNumConnectedPeers, initUint)
+	appStatusHandler.SetUInt64Value(common.MetricEpochForEconomicsData, initUint)
+
+	appStatusHandler.SetStringValue(common.MetricConsensusState, initString)
+	appStatusHandler.SetStringValue(common.MetricConsensusRoundState, initString)
+	appStatusHandler.SetStringValue(common.MetricCurrentBlockHash, initString)
+	appStatusHandler.SetStringValue(common.MetricNumConnectedPeersClassification, initString)
+	appStatusHandler.SetStringValue(common.MetricLatestTagSoftwareVersion, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PNumConnectedPeersClassification, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PPeerInfo, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PIntraShardValidators, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PIntraShardObservers, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PCrossShardValidators, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PCrossShardObservers, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PFullHistoryObservers, initString)
+	appStatusHandler.SetStringValue(common.MetricP2PUnknownPeers, initString)
+
+	appStatusHandler.SetStringValue(common.MetricInflation, initZeroString)
+	appStatusHandler.SetStringValue(common.MetricDevRewardsInEpoch, initZeroString)
+	appStatusHandler.SetStringValue(common.MetricTotalFees, initZeroString)
+
+	return nil
+}
+
+// InitConfigMetrics will init the configs from epoch config
```

Comment:
will init the enable epochs configuration metrics from epoch config

Reply:
changed